frontend/src/metabase/components/Banner/Banner.tsx

```diff
@@ -1,19 +1,48 @@
-import Markdown from "metabase/core/components/Markdown";
-import { Flex, type FlexProps } from "metabase/ui";
+import type { ReactNode } from "react";
 
-export const Banner = ({ children, ...props }: FlexProps) => {
-  const content =
-    typeof children === "string" ? <Markdown>{children}</Markdown> : children;
```

Comment:
This dance between "regular" children or the string rendered in `Markdown` is not used anywhere else. It has been used only in one random place (up until https://github.com/metabase/metabase/pull/49273, I believe).

No need to continue supporting that pattern, imo.